### ISSUE TITLE
__resources is undefined in vuex store

### ISSUE THREAD

**Issue by artofvs** [michalzaq12/electron-nuxt#42]
It looks like it's being set from resources-plugin.js which is loaded after vuex.
How can I access "__resources" from vuex module files?

edit: to clarify, my usecase is to have a sound play every time store action is performed. It should work across all pages, so store seemed like the right place.
``` javascript
store/index.js

import { Howl } from 'howler'

const notificationSound = new Howl({
  src: [`${__resources}/sound16.mp3`], //this line throws error
  loop: false
})

export const actions = {
  ...
  async fetchNewResults ({ commit }) {
    const results = await APICall()
    commit('addResults', results)
    notificationSound.play()
  }
}
```

If I should not be using resources folder this way, please lmk how to do it properly.

**Comment by michalzaq12** [comment]
Hi @artofvs ,
plugins are not intended to work in vuex modules.
The `resources-plugin.js` is loaded after vuex, as you have noticed.
You can try to import the plugin file directly into your module

```Javascript
import '../../../.electron-nuxt/renderer/resources-plugin'
```

**Comment by michalzaq12** [comment]
I will try to find a better solution and I will add it in the next version of the template

**Comment by michalzaq12** [comment]
Fixed in 1.7.0
You can use `process.resourcesPath`
